test/persistence-hooks.test.js

```diff
@@ -4,9 +4,11 @@
 // License text available at https://opensource.org/licenses/MIT
 
 'use strict';
-var should = require('./init');
-var suite = require('loopback-datasource-juggler/test/persistence-hooks.suite.js');
+describe('persistence-hooks suite test', function() {
+  before(function() {
+    require('./init.js');
+  });
 
-suite(global.getDataSource(), should, {
-  replaceOrCreateReportsNewInstance: true,
+  require('loopback-datasource-juggler/test/persistence-hooks.suite.js');
```

Comment:
And for the error
```shell
Error: done() invoked with non-Error: Error: ER_NO_DEFAULT_FOR_FIELD: Field 'id' doesn't have a default value
```

It's because these two instances in juggler are created without `id`, and mysql looks for a default value: https://github.com/strongloop/loopback-datasource-juggler/blob/master/test/persistence-hooks.suite.js#L85-L86
Confused why it complains missing default value of `id`, PK should be unique :p
My version:
```
5.7.16
MySQL Community Server (GPL)

OS: mac
```